lib/elixir/src/elixir_parser.yrl

```diff
@@ -640,6 +640,13 @@ meta_from_location({Line, Column, _}) ->
   case ?columns() of
     true -> [{line, Line}, {column, Column}];
     false -> [{line, Line}]
+  end;
+
+%% TODO: dry up
+meta_from_location({Line, Column, Indent, _}) ->
```

Comment:
I wasn't sure where exactly to put indent into the token, whether it should be in location or some other part? Should it be always added to the ast or only under a particular option like `token_metadata: true`?